### ISSUE TITLE
Intermittent Error with no code on registration

### ISSUE THREAD

**Issue by tudoramariei** [code4romania/onghub#583]
**Describe the bug**
When submitting the registration forms, some NGOs get an error.
```
Eroare
Eroare la procesarea datelor, cod: #

Încearcă din nou
```

**Steps to reproduce the behavior**
We don't have a clear way of reproducing it. The logs probably need to be checked.
The last time the issue happened was around `Jul 17, 2024, 4:21 PM`

**Comment by radulescuandrew** [comment]
@andrei-manda This was fixed in previous releases.

**Comment by andrei-manda** [comment]
@radulescuandrew it was fixed by #585
